app/src/main/res/values-sw600dp-land/refs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
> Is this the right config
I agree, I played with just `sw600dp` while writing this feature and it looked fine to me.

I used refs since I wanted to name the "normal" layout and the "tablet" layout something generic (`two_pane` and `single_pane`). Although it doesn't really matter in the context of this app since it's simple in terms of UI.